master.jinja2

```diff
@@ -21,6 +21,9 @@ timeouts:
     minutes: {% if job_timeout is defined %}{{ job_timeout }}{% else %}20{% endif %}
   connection:
     minutes: 2
+  actions:
+    finalize:
+      seconds: 60
```

Comment:
would this be needed for all jobs or only some?

and would it be an idea to make a variable for how long time we would like to wait?

Reply:
I think it makes no harm to include this in all jobs. Variable is a bad idea IMHO as this should always take under 30 seconds. 60 second timeout here means the dispatcher is struggling with LXC/docker/sth else.

Reply:
The default LAVA finalize timeout is 30sec this patch setting to 60sec.
No need of variable because it is a fixed value and since it is 60sec it will not create long waits.

We have noticed finalize timeout on LXC connected devices x15, hikey and db410c.